components/service/sync-logins/build.gradle

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion Config.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion Config.minSdkVersion
+        targetSdkVersion Config.targetSdkVersion
+    }
+
+    lintOptions {
+        warningsAsErrors true
+        abortOnError true
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            consumerProguardFiles 'proguard-rules-consumer.pro'
+        }
+    }
+}
+
+repositories {
+    ivy {
+        // TODO: This config will change once we officially publish the .aar
```

Comment:
need to ask @ncalexan if he wants to push the JCenter button to publish it on there. then we can remove this whole repo , ivy block.

Reply:
just need to be careful with the namespace. Make sure **NOT** to use the android components namespace.

Reply:
I haven't mirrored to jcenter, but they're on my bintray (and not named in this unusual manner).  `org.mozilla.sync15.logins`.